resolwe_bio/kb/views.py

```diff
@@ -108,3 +110,36 @@ def create(self, request):
             return super(FeatureViewSet, self).update(request)  # pylint: disable=no-member
         except (Feature.DoesNotExist, KeyError):  # pylint: disable=no-member
             return super(FeatureViewSet, self).create(request)  # pylint: disable=no-member
+
+
+class MappingViewSet(mixins.ListModelMixin,
+                     mixins.RetrieveModelMixin,
+                     viewsets.GenericViewSet):
+    """API view for :class:`Mapping` objects."""
+
+    serializer_class = MappingSerializer
+    filter_backends = [filters.DjangoFilterBackend]
```

Comment:
Yes, otherwise the permission filter will crash (knowledge base is permissionless). Or is there another way of disabling per-object permissions for a specific endpoint?